.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    tabWidth: 4,
+    singleQuote: true,
+    trailingComma: "es5"
```

Comment:
Let's use JSON for this file, as that's already the format for .babelrc and .eslintrc